notebook.ipynb

```diff
@@ -92,37 +92,1248 @@
     "<h2><font color=slateblue> 4. Data Exploration & Visualization </font></h2>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "71ec4af5",
+   "metadata": {},
+   "source": [
+    "<h4>Explore the meteo data<h4>"
```

Comment:
Add a title like we did in our branch
<h3><font color=steelblue>Meteo dateset </font></h3>

Reply:
I agree with Rawan's feedback on this pull request.